src/admin-views/admin-welcome-message.php

```diff
@@ -2,84 +2,213 @@
 /**
  * The template that displays the welcome message when the plugin is first activated.
  */
-?>
-
-<p class="tribe-welcome-version"><?php printf( '<strong>%1$s %2$s</strong>', esc_html__( 'Version', 'event-tickets' ), Tribe__Tickets__Main::VERSION ); ?></p>
-
-<p class="tribe-welcome-message"><?php esc_html_e( 'Event Tickets is all about getting tickets into the hands of your attendees as efficiently as possible. No muss, no fuss&mdash;just one seamless transaction on your site.', 'event-tickets' ); ?></p>
 
-<p class="tribe-welcome-message">
-<?php
-	printf(
-		'%1$s <a href="%2$s" target="_blank">%3$s <em>%4$s</em> %5$s</a> %6$s',
-		esc_html__( 'Check out the resources below for a comprehensive intro to the plugin, or head to', 'event-tickets' ),
-		admin_url( 'edit.php?post_type=tribe_events&page=tribe-common' ),
-		esc_html__( 'the', 'event-tickets' ),
-		esc_html__( 'Events', 'event-tickets' ),
-		esc_html__( 'section of the admin', 'event-tickets' ),
-		esc_html__( 'to create your very first ticket!', 'event-tickets' )
-	);
+$main = Tribe__Main::instance();
+$has_plus = class_exists( 'Tribe__Tickets_Plus__Main' );
+$has_tec = class_exists( 'Tribe__Events__Main' );
 ?>
-</p>
 
+<?php if ( $has_plus ) : ?>
+	<div class="tribe-events-admin-notice">
+		<div class="tribe-events-admin-content-wrapper">
+			<img
+				class="tribe-events-admin-notice__logo"
+				src="<?php echo esc_url( tribe_resource_url( 'images/icons/horns-white.svg', false, null, $main ) ); ?>"
+				alt="<?php esc_attr_e( 'The Events Calendar product suite logo', 'event-tickets' ); ?>"
+			/>
+			<p><strong><?php echo esc_html_x( 'WHOHOO!', 'short expression of excitement', 'event-tickets' ); ?></strong> <?php esc_html_e( 'You\'re the proud owner of Event Tickets Plus! Let\'s get started…', 'event-tickets' ); ?></p>
+		</div>
+	</div>
+<?php endif; ?>
+
+<div class="tribe-events-admin-content-wrapper tribe-events-admin-tickets <?php if ( $has_plus ) { echo 'tribe-events-admin-tickets-plus'; } ?>">
+	<img
+		class="tribe-events-admin-graphic tribe-events-admin-graphic--desktop-only"
+		<?php $desktop_graphic = $has_plus ? 'images/header/welcome-desktop-etplus.jpg' : 'images/header/welcome-desktop-et.jpg'; ?>
```

Comment:
I get why, but it would be better if these sets were above the markup near line 10

Reply:
Agreed. This (and line 34) would be great at the top of this file (after line 8 or so).